lib/metrics/statsd.js

```diff
@@ -56,15 +56,18 @@ StatsDCollector.prototype = {
   },
 
   increment: function (name, tags) {
-    var self = this
+    if (this.client) {
+      this.client.increment(
+        STATSD_PREFIX + name, 1, this.sampleRate, tags, handleErrors(this, 'increment')
+      )
+    }
+  },
 
-    if (self.client) {
-      self.client.increment(STATSD_PREFIX + name, 1, self.sampleRate, tags, function messageSentCallback(err) {
-        // this only gets called once after all messages have been sent
-        if (err) {
-          self.log.error({op: 'statsd', err: err})
-        }
-      })
+  timing: function (name, timing, tags) {
+    if (this.client) {
+      this.client.timing(
+        STATSD_PREFIX + name, timing, this.sampleRate, tags, handleErrors(this, 'timing')
```

Comment:
in the content server we made sure to add `.time` postfix to be able to tell which events are timing events. 

See https://github.com/mozilla/fxa-content-server/blob/master/server/lib/statsd-collector.js#L211

do we need this here?